Makefile.one

```diff
@@ -349,12 +365,16 @@ GO_ENV_EBPF += GOARCH=$(GO_ARCH)
 GO_ENV_EBPF += CGO_CFLAGS=$(CUSTOM_CGO_CFLAGS)
 GO_ENV_EBPF += CGO_LDFLAGS=$(CUSTOM_CGO_LDFLAGS)
 
-TRACEE_EBPF_SRC=$(shell find ./tracee-ebpf/ -type f -name '*.go' ! -name '*_test.go')
+TRACEE_EBPF_SRC_DIRS = ./cmd/tracee-ebpf/ ./tracee-ebpf/
```

Comment:
@danielpacak we "forgot" after we changed things to /cmd/. We might need to revisit the Makefile each time things are moved around as, for example, its likely we should also list ./pkg/* (this way whenever ./pkg/*.go changes the target is rebuilt).